src/main/java/org/pesmypetcare/webservice/controller/appmanager/GoogleCalendarRestController.java

```diff
@@ -0,0 +1,142 @@
+                                 @PathVariable String petName)
+        throws CalendarAccessException, DatabaseAccessException{
+        googleCalendarService.deleteSecondaryCalendar(accessToken, owner, petName);
+    }
+
+    /**
+     * Returns all Calendar Events from a specified Calendar.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @return List containing all the Events from the specified Calendar
+     * @throws CalendarAccessException If an error occurs when accessing the calendar
+     * @throws DatabaseAccessException If an error occurs when accessing the database
+     */
+    @GetMapping("/{owner}/{petName}")
+    public List<Event> getAllEventsFromCalendar(@RequestHeader(TOKEN) String accessToken, @PathVariable String owner,
+                                         @PathVariable String petName)
+        throws CalendarAccessException, DatabaseAccessException{
+        return googleCalendarService.getAllEventsFromCalendar(accessToken, owner, petName);
+    }
+
+    /**
+     * Creates an Event in a specified Google Calendar.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @param eventEntity Event to create
+     * @throws CalendarAccessException If an error occurs when accessing the calendar
+     * @throws DatabaseAccessException If an error occurs when accessing the database
+     */
+    @PostMapping("/event/{owner}/{petName}")
+    public void createEvent(@RequestHeader(TOKEN) String accessToken, @PathVariable String owner,
+                     @PathVariable String petName, @RequestBody EventEntity eventEntity)
+        throws CalendarAccessException, DatabaseAccessException{
+        googleCalendarService.createEvent(accessToken, owner, petName, eventEntity);
+    }
+
+    /**
+     * Retrieves an Event in a specified Google Calendar.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @param body Body of the request containing the id of the event to retrieve with key eventId assigned
+     * @return Event retrieved
+     * @throws CalendarAccessException If an error occurs when accessing the calendar
+     * @throws DatabaseAccessException If an error occurs when accessing the database
+     */
+    @GetMapping("/event/{owner}/{petName}")
+    public EventEntity retrieveEvent(@RequestHeader(TOKEN) String accessToken, @PathVariable String owner,
+                              @PathVariable String petName, @RequestBody Map<String, Object> body)
+        throws CalendarAccessException, DatabaseAccessException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)